client/src/docs-ui/search-bar/search-bar.tsx

```diff
@@ -0,0 +1,42 @@
+import {Component, Host, h, Build, Listen} from "@stencil/core";
+import {searchStyle} from "./search-bar.style";
+import {transformData} from "../../utils/transform-search-data";
+import {ALGOLIA_API_KEY, ALGOLIA_INDEX_NAME} from "../../constants/algolia";
+
+@Component({tag: "docs-search-bar", shadow: false})
+export class DocsSearchBar {
+  initDocSearch() {
+    if (Build.isBrowser) {
+      // @ts-ignore
+      docsearch({
+        apiKey: ALGOLIA_API_KEY,
+        indexName: ALGOLIA_INDEX_NAME,
+        inputSelector: ".amplify-docs-search-input:not(.ds-input)",
```

Comment:
Curious why this changed from `"#amplify-docs-search-input"`

Reply:
So that we never re-initialize an input as an Algolia docsearch element